### PR TITLE
Dynamic betweenness centrality bugfix

### PR DESCRIPTION
**Pull request by antepusic**
### Description

Fix the behavior of `betweenness_centrality_online` that can cause the following bugs:
* module’s algorithm staying initialized after reload
* no nodes have their BC scores updated after edge update

### Pull request type

- [x] Bugfix
- [ ] Algorithm/Module
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### Related issues

Delete if this PR doesn't resolve any issues. Link the issue if it does.

######################################

#### Module/Algorithm
- [ ] Core algorithm/module implementation
- [ ] [Query module](https://memgraph.com/docs/memgraph/reference-guide/query-modules) implementation
- [ ] Unit tests
- [ ] End-to-end tests
- [ ] Code documentation
- [ ] README short description
- [ ] Documentation on [memgraph/docs](https://github.com/memgraph/docs)

######################################

### CODE REVIEW

File: e2e/betweenness_centrality_online_test/test_online_get/input.cyp [hunk 1]
```diff
@@ -7,6 +7,7 @@ queries:
         MERGE (a: Node {id: 3}) MERGE (b: Node {id: 4}) CREATE (a)-[:RELATION]->(b);
         MERGE (a: Node {id: 3}) MERGE (b: Node {id: 5}) CREATE (a)-[:RELATION]->(b);
         MERGE (a: Node {id: 4}) MERGE (b: Node {id: 5}) CREATE (a)-[:RELATION]->(b);
+        CALL betweenness_centrality_online.set() YIELD *;
```
Comment (jmatak) [inline e2e/betweenness_centrality_online_test/test_online_get/input.cyp:10]: What happens if `set` is not called here

Comment (antepusic) [reply e2e/betweenness_centrality_online_test/test_online_get/input.cyp:10]: When `get` is called without there being previously computed BC scores, it defaults to computing them with `set` using the default parameters.
See `test_get_uninitialized` for an example.

Comment (jmatak) [reply e2e/betweenness_centrality_online_test/test_online_get/input.cyp:10]: What happens if update is called on uninitialized graph?

Comment (antepusic) [reply e2e/betweenness_centrality_online_test/test_online_get/input.cyp:10]: Same thing ([test_online_update_uninitialized](https://github.com/memgraph/mage/tree/T261-MAGE_dynamic-BC-bug/e2e/betweenness_centrality_online_test/test_online_update_uninitialized)).

**Review by jmatak — APPROVED** [review]

**Pull request merged**